source/appModules/securecrt.py

```diff
@@ -2,20 +2,25 @@
 #A part of NonVisual Desktop Access (NVDA)
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
-#Copyright (C) 2010-2012 NV Access Limited
+#Copyright (C) 2010-2017 NV Access Limited, , Noel Romey
 
 """App module for SecureCRT
 """
 
+import re
 import oleacc
 from NVDAObjects.behaviors import Terminal
 from NVDAObjects.window import DisplayModelEditableText, DisplayModelLiveText
 import appModuleHandler
 
+# Regexp which matches the terminal window class in all existing versions
+# (and hopefully future ones).
+RE_TERMINAL_WINCLASS = re.compile ("AfxFrameOrView\d{2,}u")

```

Comment:
I think it would be helpful to give some examples for the kind of values that we are matching. Luckily we used to test against a set of examples `"AfxFrameOrView80u", "AfxFrameOrView90u", "AfxFrameOrView100u"`